pkg/service/backup/restore_worker_index.go

```diff
@@ -194,11 +194,27 @@ func (w *indexWorker) workFunc(ctx context.Context, run *RestoreRun, target Rest
 
 			pr.setRestoreCompletedAt()
 			w.insertRunProgress(ctx, pr)
-
 			restoredBytes := pr.Downloaded + pr.Skipped + pr.VersionedProgress
 			w.metrics.DecreaseRemainingBytes(w.ClusterID, target.SnapshotTag, w.location, w.miwc.DC, w.miwc.NodeID,
 				pr.Keyspace, pr.Table, restoredBytes)
 
+			// Calculate restore progress in percents
+			var progress float64
+			if fm.Size == 0 {
+				progress = 100
+			} else {
+				progress = float64(restoredBytes) / float64(fm.Size) * 100
```

Comment:
@charconstpointer this formula won't work.

`restoredBytes` defines how much of the data (in the context of location, dc, node, keyspace and table) has been restored in the **current** chunk of the restore procedure.
Chunk is just a couple of SSTable files sent to a particular node that calls load&stream later on.

`fm.Size` defines the size of all SSTables from the location.